tests/tests_integration/test_file_conflicts_pathtable.py

```diff
@@ -0,0 +1,38 @@
+# flake8: noqa
+# fmt: off
+
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_pathtable(base_dir):
```

Comment:
Would it be easier to store this in a csv or something? It's hard to parse as code.